### DISCUSSION TITLE
Scroller: dynamic size for itemSize

### DISCUSSION THREAD

**Discussion by ayangabryl** [primefaces/community#2290] [PrimeNG]
is there a way to use the p-scroller for like message app? each content has different sizes or height. Is it possible in scroller now or is it not?

**Comment by Max0nyshchenko** [comment]
As far as I know, it is not supported. So I prepared the draft PR with support of dynamic size for itemSize - https://github.com/primefaces/primeng/pull/16648

**Comment by DanniKach** [comment]
Our team is really needed and waiting for this functionality. Our app has design required expanded items with infinite scroll, which was impossible in current prime ng. So hope it will be approved and merged as soon as possible.
